istioctl/cmd/auth.go

```diff
@@ -227,4 +269,6 @@ func init() {
 		"Authorization policy file")
 	upgradeCmd.PersistentFlags().StringSliceVarP(&serviceFiles, "service", "s", []string{},
 		"Kubernetes Service resource that provides the mapping relationship between service name and pod labels")
+	validatorCmd.PersistentFlags().StringSliceVarP(&policyFiles, "file", "f", []string{},
```

Comment:
The regular validate uses `flags.StringSliceVarP(&filenames, "filename", "f", nil, "Names of files to validate")`

Reply:
Wasn't trying to align with `istioctl validate`. This PR follows existing conventions in `auth.go`.